tests/test_wikipedia_toast_button.py

```diff
@@ -0,0 +1,19 @@
+import time
+import allure
+import pytest
+from selenium.webdriver import Keys
+from selenium.webdriver.chrome import webdriver
+@allure.title("Return to the main page using toast menu")
+def test_search_from_main_page(pages):
+    with allure.step("Search for article"):
+        pages.wiki_home.search_bar.send_keys("Terminator")
+        with allure.step("click enter"):
+            pages.wiki_home.search_bar.send_keys(Keys.ENTER)
+        time.sleep(4)
```

Comment:
Let's try to resolve this issue without using `time.sleep(4)` ;) Can you please try to use `WebDriverWait` or a retry function (you can use the one(s) we wrote during the course)?

Reply:
I believe you'll have to apply the wait or the retry to searching the main header and getting text from it. For retry it would look something like this: 
```
assert retry_until_not_fails(pages.wiki_article.main_header.text) == "Terminator"
```